designer/client/page-edit.js

```diff
@@ -305,13 +308,15 @@ export class PageEdit extends React.Component {
           <button className="govuk-button" type="submit">
             {i18n("save")}
           </button>{" "}
-          <button
-            className="govuk-button"
-            type="button"
-            onClick={this.onClickDuplicate}
-          >
-            {i18n("duplicate")}
-          </button>{" "}
+          <FeatureToggle feature={FEATURE_EDIT_PAGE_DUPLICATE_BUTTON}>
```

Comment:
I like this API

Reply:
maybe we should use an enum? {Feature.EDIT_PAGE_DUPLICATE_BUTTON} or a type with literal strings? for nicer code completion. Fine for now - we only have one flag

Reply:
So I was using types and I switched to enums, I think they're same same really. Not much difference between it.